joblib/_dask.py

```diff
@@ -124,7 +124,7 @@ def __call__(self, tasks=None):
         with parallel_backend('dask'):
             for func, args, kwargs in tasks:
                 results.append(func(*args, **kwargs))
-        return results
+            return results
```

Comment:
Please add an inline comment to explain why it's important to return within the context manager because it's not obvious.

Reply:
@fcharras do you remember why we needed this?

Reply:
no.. :thinking: the diff on this file looks meaningless. At least it's harmless.